docs/MO_DG/prepare_model/Config_Model_Optimizer.md

```diff
@@ -375,22 +375,14 @@ install_prerequisites_caffe.sh
 ```
 install_prerequisites_caffe.bat
 ```
-*   For TensorFlow\* 1.x on Linux:
+*   For TensorFlow\* on Linux:
 ```
 install_prerequisites_tf.sh
 ```
-*   For TensorFlow 1.x on Windows:
+*   For TensorFlow on Windows:
```

Comment:
TensorFlow\*

Reply:
Looking to this file we don't have `*` after all frameworks names on windows and have it after all frameworks names on linux. I don't know the reason, but If you think it is necessary then we need to add `*` to all frameworks names on windows. What do you think?

Reply:
@tsavina please help

Reply:
@tsavina Please, take a look.